.github/workflows/qualitycheck.yml

```diff
@@ -27,11 +27,7 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        python -m pip install black pylama
-    - name: Black
-      run: |
-        black --diff --color --line-length 119 thermal_base
-        black --check --line-length 119 thermal_base
+        python -m pip install pylama
```

Comment:
Was black causing issues?

Reply:
Yeah. Black and pylama sorted imports differently